bindings/wasm/src/did/wasm_document.rs

```diff
@@ -87,10 +98,84 @@ impl WasmDocument {
     WasmDID(self.0.id().clone())
   }
 
+  /// Returns a list of the document controllers.
+  #[wasm_bindgen(js_name = controller)]
+  pub fn controller(&self) -> DIDArray {
+    match self.0.controller() {
+      Some(controllers) => controllers
+        .iter()
+        .cloned()
+        .map(WasmDID::from)
+        .map(JsValue::from)
+        .collect::<js_sys::Array>()
+        .unchecked_into::<DIDArray>(),
+      None => js_sys::Array::new().unchecked_into::<DIDArray>(),
+    }
+  }
+
+  /// Returns a set of the document's `alsoKnownAs`.
+  #[wasm_bindgen(js_name = alsoKnownAs)]
+  pub fn also_known_as(&self) -> DIDArray {
```

Comment:
The `controller` and `alsoKnownAs` fields need setters as well to be equivalent to Rust. We should also consider returning None/null in the accessors if they are not present, since that's more accurate?

Reply:
In Rust it doesn't return an `Option` so I kept the same behavior. But an empty array is not wrong here in my opinion.

Reply:
The `controller` returns an `Option`, just not `alsoKnownAs`. Returning an empty array should be fine, but we should allow taking `null` for the setters to unset them?